generators/add-web-assets/index.js

```diff
@@ -52,25 +52,30 @@ class AddWebAssets extends Generator {
   }
 
   async prompting () {
-    const webAssetsGenerator = excReactWebAssetsGenerator
-
-    // code for later -> as of now support only exc-react
-    // if (!this.options['skip-prompt']) {
-    //   const promptProps = await this.prompt([
-    //     {
-    //       // for now we just have one webAsset generator
-    //       type: 'list',
-    //       name: 'webAssetsGenerator',
-    //       message: 'Which type of UI do you want to add to your project?\nselect template to generate',
-    //       choices: [{ name: 'Adobe Experience Cloud Shell - React', value: excReactWebAssetsGenerator }, { name: 'Raw HTML/JS', value: rawWebAssetsGenerator }],
-    //       validate: utils.atLeastOne
-    //     }
-    //   ])
-    //   webAssetsGenerator = promptProps.webAssetsGenerator
-    // }
-
-    // run ui generator
-    this.composeWith(webAssetsGenerator, this.options)
+    if (!this.options['skip-prompt']) {
+      const promptProps = await this.prompt([
+        {
+          type: 'list',
+          name: 'webAssetsGenerator',
+          message: 'Which type of UI do you want to add to your project?\nselect template to generate',
+          choices: [
+            {
+              name: 'Adobe Experience Cloud Shell - React',
+              value: excReactWebAssetsGenerator
+            },
+            {
+              name: 'Raw HTML/JS',
```

Comment:
Raw HTML/JS UI

(Please note that this UI will also be able to deploy within Adobe Experience Cloud, so it should also integrate with the ExC Shell client-side API).